server/service/src/sync/sync_status/logger.rs

```diff
@@ -232,9 +232,14 @@ impl SyncLogError {
             // Sync Api Error
             SyncError::CentralPullError(CentralPullError::SyncApiError(error))
             | SyncError::RemotePullError(RemotePullError::SyncApiError(error))
-            | SyncError::PostInitialisationError(PostInitialisationError(error))
+            | SyncError::PostInitialisationError(PostInitialisationError::SyncApiError(error))
+            | SyncError::PostInitialisationError(
```

Comment:
I'll link this comment to the core issue:

TODO we should have: SyncLogRowErrorCode::InitialisaitonTimeoutReached similar to SyncLogRowErrorCode::IntegrationTimeoutReached